hangman.rb

```diff
@@ -0,0 +1,113 @@
+class Hangman
+
+  def initialize
+    @answer = pick_answer
+    @no_of_guesses = 0
+    @blanks_array = answer_to_blanks
+
+    start
+    puts @answer #temporary!!
+    art
+    get_guess
+
+  end
+
+  def start
+    puts "Welcome to Hangman!"
+  end
+
+  def pick_answer
+    # randomly pick the answer from the library
+    library = ["cat", "dog", "mouse", "letter", "elephant", "bottomless", "house", "polecat", "coffee", "apple"]
+    @answer = library.sample
+
+    return @answer
+  end
+
+  def answer_to_blanks
+    # convert the answer to the number of blanks of the chosen word
+    num_of_blanks = @answer.length
+    @blanks = "_" * num_of_blanks
+    puts @blanks
```

Comment:
Probably wouldn't want this `puts` here in favor of putting it in the `art` method